jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -540,6 +542,33 @@ private boolean isDDLCommand(int updatedCount, int columnCount) throws SQLExcept
     return queries;
   }
 
+  private void executePreCode(Connection connection) {
+    String precode = getProperty(ZEPPELIN_PRECODE_PROPERTY_KEY);
+    if (StringUtils.isNotEmpty(precode)) {
+      logger.info("Run SQL precode '{}'", precode);
+      try {
```

Comment:
`Connection` and `Statement` are `AutoCloseable`. Isn't it better to use try-with-resources?